dom0-updates/qubes-dom0-update

```diff
@@ -336,6 +366,36 @@ else
     fi
 fi
 
+# Switching audio
+
+if [ "$AUDIO_SWITCH" = "pipewire" ]; then
+    if ! rpm -q pipewire pipewire-pulseaudio >/dev/null; then
+        echo "Switching from Pulseaudio to PipeWire" >&2
+        SWITCHING_AUDIO_IN_PROGRESS=yes "$0" "${UPDATEVM_OPTS[@]}" --action=install --allowerasing pipewire pipewire-pulseaudio || exit $?
+        echo "Audio daemon switched to PipeWare, you can undo it with qubes-domo0-update --switch-audio-server-to=pulseaudio" >&2
+        # do not leave the user with stopped both daemons
+        user=$(getent group qubes|cut -f 4 -d :|cut -f 1 -d ,) || exit 1
+        uid=$(id -u "$user") || exit 1
+        sudo -u "$user" XDG_RUNTIME_DIR="/run/user/$uid" systemctl --user start pipewire-pulse
+    else
+        echo "PipeWire already installed" >&2
+    fi
+    echo 1 > /var/lib/qubes/.audio-switch-done
+elif [ "$AUDIO_SWITCH" = "pulseaudio" ]; then
+    if ! rpm -q pulseaudio >/dev/null; then
+        echo "Switching from PipeWire to Pulseaudio" >&2
+        SWITCHING_AUDIO_IN_PROGRESS=yes "$0" "${UPDATEVM_OPTS[@]}" --allowerasing --action=swap pipewire pulseaudio || exit $?
+        echo "Audio daemon switched to Pulseaudio, you can undo it with qubes-domo0-update --switch-audio-server-to=pipewire" >&2
+        # do not leave the user with stopped both daemons
+        user=$(getent group qubes|cut -f 4 -d :|cut -f 1 -d ,) || exit 1
+        uid=$(id -u "$user") || exit 1
+        sudo -u "$user" XDG_RUNTIME_DIR="/run/user/$uid" systemctl --user stop pipewire pipewire-pulse pipewire.socket pipewire-pulse.socket
+        sudo -u "$user" XDG_RUNTIME_DIR="/run/user/$uid" systemctl --user start pulseaudio
+    else
+        echo "Pulseaudio already installed" >&2
+    fi
```

Comment:
```suggestion
    fi
    echo 1 > /var/lib/qubes/.audio-switch-done
```

This covers the case where `--switch-audio-server-to=pulseaudio` is passed the first time after `qubes-dom0-update` is updated.  Otherwise the user would still get switched to PipeWire the next time they update, which is wrong.